src/debugAdapter/goDebug.ts

```diff
@@ -276,162 +302,193 @@ class Delve {
+								runArgs.push(launchArgs.buildFlags);
+							}
+							runArgs.push(program);
+							if (launchArgs.args) {
+								runArgs.push(...launchArgs.args);
+							}
+							this.debugProcess = spawn(getBinPathWithPreferredGopath('go', []), runArgs, { env });
+							this.debugProcess.stderr.on('data', chunk => {
+								let str = chunk.toString();
+								if (this.onstderr) { this.onstderr(str); }
+							});
+							this.debugProcess.stdout.on('data', chunk => {
+								let str = chunk.toString();
+								if (this.onstdout) { this.onstdout(str); }
+							});
+							this.debugProcess.on('close', (code) => {
+								logError('Process exiting with code: ' + code);
+								if (this.onclose) { this.onclose(code); }
+							});
+							this.debugProcess.on('error', (err) => {
+								reject(err);
+							});
+							resolve();
+							return;
 						}
-						this.debugProcess = spawn(getBinPathWithPreferredGopath('go', []), runArgs, { env });
-						this.debugProcess.stderr.on('data', chunk => {
-							let str = chunk.toString();
-							if (this.onstderr) { this.onstderr(str); }
-						});
-						this.debugProcess.stdout.on('data', chunk => {
-							let str = chunk.toString();
-							if (this.onstdout) { this.onstdout(str); }
-						});
-						this.debugProcess.on('close', (code) => {
-							logError('Process exiting with code: ' + code);
-							if (this.onclose) { this.onclose(code); }
-						});
-						this.debugProcess.on('error', (err) => {
-							reject(err);
-						});
-						resolve();
-						return;
 					}
 				}
-			}
-			this.noDebug = false;
-			let serverRunning = false;
+				this.noDebug = false;
+
+				// Get default LoadConfig values according to delve API:
+				// https://github.com/derekparker/delve/blob/c5c41f635244a22d93771def1c31cf1e0e9a2e63/service/rpc1/server.go#L13
+				// https://github.com/derekparker/delve/blob/c5c41f635244a22d93771def1c31cf1e0e9a2e63/service/rpc2/server.go#L423
+				this.loadConfig = launchArgs.dlvLoadConfig || {
+					followPointers: true,
+					maxVariableRecurse: 1,
+					maxStringLen: 64,
+					maxArrayValues: 64,
+					maxStructFields: -1
+				};
 
-			// Get default LoadConfig values according to delve API:
-			// https://github.com/derekparker/delve/blob/c5c41f635244a22d93771def1c31cf1e0e9a2e63/service/rpc1/server.go#L13
-			// https://github.com/derekparker/delve/blob/c5c41f635244a22d93771def1c31cf1e0e9a2e63/service/rpc2/server.go#L423
-			this.loadConfig = launchArgs.dlvLoadConfig || {
-				followPointers: true,
-				maxVariableRecurse: 1,
-				maxStringLen: 64,
-				maxArrayValues: 64,
-				maxStructFields: -1
-			};
+				if (!existsSync(launchArgs.dlvToolPath)) {
+					log(`Couldn't find dlv at the Go tools path, ${process.env['GOPATH']}${env['GOPATH'] ? ', ' + env['GOPATH'] : ''} or ${envPath}`);
+					return reject(`Cannot find Delve debugger. Install from https://github.com/derekparker/delve & ensure it is in your Go tools path, "GOPATH/bin" or "PATH".`);
```

Comment:
I think the path is now `https://github.com/go-delve/delve/`

Reply:
Fixed all instances.